lib/item_stream_handler.rb

```diff
@@ -19,6 +19,7 @@ def add_checkout(checkout)
     # Add checkout to end:
     @checkouts << checkout
 
+    Application.logger.debug "ItemStreamHandler#handle: Added checkout: #{checkout.id} (#{checkout.barcode}) \"#{checkout.title}\""
```

Comment:
Comment prefix is a tad misleading. Should begin "ItemStreamHandler#add_checkout: ..."

Reply:
Fixed!